### ISSUE TITLE
OSX accessibility zoom features crash brave

### ISSUE THREAD

**Issue by cazssiew** [brave/brave-browser#8149]
## Description 
Using the OSX zoom features (ctrl+scroll or ctrl+hover text) crashes brave

## Steps to Reproduce
   1. enable zoom and large hover text in system preferences > accessibility > zoom
   2. use in brave according to the selected hotkeys

## Actual result:
brave crashes

## Expected result:
zooming in/out / large hover text 

## Reproduces how often: 
most often crashes instantly, occasionally takes a few seconds, but always ends up crashing.

## Brave version (brave://version info)
Brave | 1.3.113 Chromium: 80.0.3987.87 (Official Build) (64-bit)
Revision | 449cb163497b70dbf98d389f54e38e85d4c59b43-refs/branch-heads/3987@{#801}
OS | macOS Version 10.15.3 (Build 19D76)

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? don't know
- Can you reproduce this issue with the dev channel? don't know
- Can you reproduce this issue with the nightly channel? don't know

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? no
- Does the issue resolve itself when disabling Brave Rewards? no
- Is the issue reproducible on the latest version of Chrome? no

**Comment by rebron** [comment]
Easily reproduces on macOS Catalina. Not an issue on Mojave.

**Comment by kjozwiak** [comment]
Thanks for the report @cazssiew 👍 Much appreciated. We'll make sure this gets addressed/fixed in the next release which is typically when we have a minor chromium bump.

**Comment by cazssiew** [comment]
Awesome, thanks! Unfortunately I only started using the macos zoom very recently so I'm not sure. Sorry I can't be of more help.

**Comment by rebron** [comment]
Reproducible on `1.2.43` using a fresh profile.

**Comment by mkarolin** [comment]
@rebron could you, please, try with the Nightly Version 1.11.31 (Chromium 83). It didn't crash for me with `cr83` branch build.

**Comment by Brave-Matt** [comment]
+1 from Community:
https://community.brave.com/t/brave-crashes-when-hover-text-is-locked-in-macos-catalina/202569/3 

@rebron can we get an update on this? I'm sure there are other users seeing this as well.

**Comment by iefremov** [comment]
@Brave-Matt Should be fixed in Nighlty with https://github.com/brave/brave-core/pull/7832

**Comment by iefremov** [comment]
The fix brave/brave-core#7832 was uplifted to beta and release

**Comment by LaurenWags** [comment]
@iefremov while testing I saw the app crash when attempting to reproduce using 1.19.x. Using 1.20.104 the app doesn't crash anymore but it seems crash reports are still being generated (quite generously) in brave://crashes. Examples:
94070000-ab7a-e305-0000-000000000000
92070000-ab7a-e305-0000-000000000000
95070000-ab7a-e305-0000-000000000000
91070000-ab7a-e305-0000-000000000000

**Comment by iefremov** [comment]
this is expected @LaurenWags , all good.

**Comment by LaurenWags** [comment]
Verified passed with `macOS x64 10.15.7 (Catalina)`

```
Brave | 1.20.104 Chromium: 88.0.4324.152 (Official Build) (x86_64)
-- | --
Revision | 6579930fc53b4dc589c042bec9d0a3778326974d-refs/branch-heads/4324@{#2106}
OS | macOS Version 10.15.7 (Build 19H512)
```

Verified using STR from https://github.com/brave/brave-browser/issues/8149#issue-562225385. No app crashes observed when using keyboard shortcuts for zoom in/out and large hover text. Note, crash reports observed as noted in https://github.com/brave/brave-browser/issues/8149#issuecomment-781373685.

----------------
